### ISSUE TITLE
Dry run flag that doesn't do anything in the remote host

### ISSUE THREAD

**Issue by CristhianMotoche** [stackbuilders/hapistrano#226]
Let's include a flag that doesn't do any action on the server. It will be a dry run that will print the commands that hapistrano will run. If the commands are okay then a user can proceed with actually running without that flag.

```
$ hap deploy --dry-run

[INFO] git clone ...
[INFO] stack build ...
...

[INFO] Finished (DRY RUN: Commands were executed)
```

**Comment by m-eraf** [comment]
i want to contribute to this
plz assign

**Comment by CristhianMotoche** [comment]
Thanks for your eagerness to contribute to hapistrano. @m-eraf I assigned it to you.

**Comment by m-eraf** [comment]
I am little bit new to Haskell will u plz guide me to run the project..

**Comment by CristhianMotoche** [comment]
Sure thing.

First, install the Haskell environment in your local machine. I recommend installing it with [ghcup](https://www.haskell.org/ghcup/).
Second, install GHC 9.0 and Cabal 3.10.0.0. Those are the versions that we use in GH Actions for building and testing hapistrano.
Third, follow the [steps in the readme](https://github.com/stackbuilders/hapistrano#getting-started) which you can also find in the [CI settings](https://www.haskell.org/ghcup/) to build and test the project.

I hope that helps to get you started.

**Comment by m-eraf** [comment]
Thanks...
I will get started with Haskell and the project right away.

**Comment by CristhianMotoche** [comment]
Solved in #246
